Restructured `Payments` chapter. (#625)
BitcoinDesign/Guide: guide/payments/units-and-symbols.md

   height = 198
%}

Complex use cases often require users to interact with addresses, public keys, and other encrypted data, where it is important that each character is easily identifiable. Slashed zeros can help distinguish 0 (the number zero) and O (the uppercase letter "O"), which can look very similar in some typefaces.

Note that uppercase letter "O", uppercase letter "I", lowercase letter "l", and the number "0" are not valid characters in addresses, in order to prevent mistakes. As users may not be aware of this, using slashed zeros may still be a helpful design choice.

---

> Next we to a technical deep dive into out [transactions]({{ '/guide/payments/transaction' | relative_url }}) on bitcoin work.

Reviewer: ```suggestion
Next, we do a technical deep dive into how [transactions]({{ '/guide/payments/transaction' | relative_url }}) on bitcoin work.
```

{% include next-previous.html
   previousUrl = "/guide/payments/privacy"
   previousName = "Transaction privacy"
   nextUrl = "/guide/Payments/transactions"

>    nextName = "Transaction overview"

Reviewer: ```suggestion
   nextName = "Transactions overview"
```
